JASP-Engine/JASP/R/LScommon.R

```diff
@@ -114,8 +114,7 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
   gaps <- which(diff(indices) > 1)
   if(length(gaps) > 0 && !allowSplit) {
     # In this case, return shortest 95% CrI
-    warning("The HDI is discontinuous but allowSplit = FALSE;
```

Comment:
@Kucharssim is it fine to remove this warning, or do you catch it somewhere to show as a footnote?

Reply:
Sorry, I did not see the mention - this is not my file! 😄